libraries/APM_Control/AP_AutoTune.h

```diff
@@ -18,9 +18,9 @@ class AP_AutoTune
     };
 
     enum ATType {
-        AUTOTUNE_ROLL  = 0,
-        AUTOTUNE_PITCH = 1,
-        AUTOTUNE_YAW = 2,
+        AUTOTUNE_ROLL  = 1,
+        AUTOTUNE_PITCH = 2,
+        AUTOTUNE_YAW = 4,
```

Comment:
Shouldn't need to change anything in here

Same conversions need to happen when executing the mission item as we do when we run the mavlink command.